index.html

```diff
@@ -9,148 +10,245 @@
+            <article class="languages">
+                <img src="./Assets/Icons.png" alt="">
+                <h1>Languages</h1>
+                <ul class="lang">
+                    <li class="lr"><a>Ruby on rails</a></li>
+                    <li class="lc"><a>CSS</a></li>
+                    <li class="lj"><a>JavaScript</a></li>
+                    <li class="lh"><a>html</a></li>
+                </ul>
+
+            </article>
+            <article class="frame">
+                <img src="./Assets/Icons (1).png" alt="">
+                <h1>Frameworks</h1>
+                <ul class="fr">
+                    <li class="lc"><a>Bootstrap</a></li>
+                    <li class="lr"><a>Ruby on rails</a></li>
+                    <li class="lj"><a>RSpec</a></li>
+                    <!-- <li class="lh"><a>Capypara</a></li> -->
+                    <!-- <li class="lh"><a>Selenium</a></li> -->
+                </ul>
+
+            </article>
+            <article class="frame">
+                <img src="./Assets/Icons (2).png" alt="">
+                <h1>Skills</h1>
+                <ul class="fr">
+                    <li class="lc"><a>Bootstrap</a></li>
+                    <li class="lr"><a>Ruby on rails</a></li>
+                    <li class="lj"><a>RSpec</a></li>
+                    <li class="lh"><a>Capypara</a></li>
+                    <li class="lh"><a>Selenium</a></li>
+                </ul>
+
+            </article>
+        </section>
+        <section class="endp">
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+
+
+            <section class="contact_form">
+                <h3>Contact me</h3>
+                <form action="https://formspree.io/f/myyvrwdd" method="POST">
+
+                    <label for="first_name"></label>
+                    <input name="first_name" type="text" required placeholder="Full Name" maxlength="30"
+                        minlength="2" />
+                    <br>
+
+                    <label for="email"></label>
+                    <input name="email" type="email" required placeholder="Email Address" />
+
+                    <label for="message"></label>
+                    <textarea name="message" cols="30" rows="10" placeholder="Enter your message here ..." required
+                        maxlength="500" minlength="2"> Enter your Messege here!</textarea>
+                    <br>
+                    <button type="submit" class="getintouch">Get in touch</button>
+
+                </form>
+            </section>
+        </section>
         <ul class="social_form">
             <li><a href="#" class="fa fa-facebook-square"></a></li>
             <li><a href="#" class="fa fa-twitter-square"></a></li>
             <li><a href="#" class="fa fa-github-square"></a></li>
             <li><a href="#" class="fa fa-instagram"></a></li>
         </ul>
         <h3 class="eaddress">anas599@yahoo.com</h3>
```

Comment:
- I think the space/gaps between your form, social icons and email is too much, can we try reducing them to look like that on figma?

Here is what i can see currently on my browser. 👇🏽 

![exsp](https://user-images.githubusercontent.com/99246359/206032601-59c42576-fb46-480d-a89d-25ac2e023032.PNG)